backend/lucene/src/main/java/org/hibernate/search/backend/lucene/document/model/impl/LuceneIndexSchemaNamedPredicateNode.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.backend.lucene.document.model.impl;
+
+public class LuceneIndexSchemaNamedPredicateNode<F> {
```

Comment:
Type parameter `<F>` can be removed here and replaced with `NamedPredicateFactory` everywhere.